packages/coreutils/src/time.ts

```diff
@@ -17,9 +17,7 @@ const UNITS: { name: Intl.RelativeTimeFormatUnit; milliseconds: number }[] = [
  * The namespace for date functions.
  */
 export namespace Time {
-  // Intl.RelativeTimeFormatStyle contains these, but it requires `ES2020.Intl`.
-  // We currently compile to an `ES2018` target.
-  export type HumanStyle = 'long' | 'short' | 'narrow';
+  export type HumanStyle = Intl.ResolvedRelativeTimeFormatOptions['style'];
```

Comment:
As the original author of the code you're replacing, I'm OK with deleting `HumanStyle` entirely, replacing references to `Time.HumanStyle` with `Intl.RelativeTimeFormatStyle`.

Reply:
Although recent, the commit that introduced the `format` parameter did make it into coreutils 6.2.0, and removing this exported type would _technically_ be a breaking change to the coreutils library. It becomes a redundant alias, for sure, but equally it doesn't do any harm to keep it.

As an aside, where the type is used elsewhere in jupyterlab, `Time.HumanStyle` probably gives clearer context than `Intl.Blah`, although YMMV.

Reply:
I'm OK to keep `Time.HumanStyle` as well; I like the idea of avoiding a breaking change if we can help it. Thanks again for working on this!